mss/windows.py

```diff
@@ -174,6 +178,17 @@ def _set_dpi_awareness(self):
             # Windows Vista, 7, 8 and Server 2012
             self.user32.SetProcessDPIAware()
 
+    def _get_srcdc(self):
+        # Fix thread unsafe issue:
+        # In multithreading, if the thread who creates srcdc is died, srcdc is no longer valid to grab screen.
+        # The srcdc attribute is replaced with srcdc_dict to maintain the srcdc values in multithreading
+        # Since current thread and main thread is always alive, reuse their srcdc value first.
```

Comment:
```suggestion
        """
        Retrieve a thread-safe HDC from GetWindowDC().
        In multithreading, if the thread who creates *srcdc* is dead, *srcdc* will
        no longer be valid to grab the screen. The *srcdc* attribute is replaced
        with *srcdc_dict* to maintain the *srcdc* values in multithreading.
        Since the current thread and main thread are always alive, reuse their *srcdc* value first.
        """
```